api-examples/rbacv1/main.py

```diff
@@ -0,0 +1,324 @@
+        :param user_role_pairs: a list of (user_id_v2, role_id_or_alias) to know users and capacity to add to team.
+        :return: a dictionary of the json response from the call.
+        """
+        add_to_team = []
+        for user_id, role in user_role_pairs:
+            add_to_team.append({"user_id_v2": user_id,
+                                "team_role": role})
+        version = self.current_team_version(team)
+        payload = {
+            "version": version,
+            "add_team_membership": [
+                add_to_team
+            ]
+        }
+        self._do_put("rbac/teams/{team}".format(team=team), payload)
+
+
+class CSVUser:
+    """
+    CSVUser represents user information as present in each row of the sample csv.
+    """
+
+    def __init__(self, email="", team="", orgrole="", teamrole=""):
+        self.email = email
+        self.team = team
+        self.organization_role = orgrole  # NOTE: These values are not yet stable and might change.
+        self.team_role = teamrole  # NOTE: These values are not yet stable and might change.
+
+
+def main():
+    api_v4 = SLAPIClient(SHIFTLEFT_ACCESS_TOKEN, SHIFTLEFT_ORG_ID)
+
+    teams = api_v4.list_teams()
+    users = api_v4.list_users()
+
+    add_to_teams = {}
+
+    with open("rbac.csv", "r") as csv_file:
+        csv_reader = csv.DictReader(csv_file)
+        for row in csv_reader:
+            # Read one user from CSV
+            user = CSVUser(**row)
+            # Create the team this user should belong to if it doesn't exist
+            if user.team in teams:
+                print("Team {} exists for this organization".format(user.team))
+            else:
+                print("Team '{}' does not exist for this organization,"
+                      " we will create it and assign '{}'".format(user.team, user.email))
+                teams.append(api_v4.create_team(user.team))
+
+            # Assign the user organization wide role.
+            user_id = users.id_for_email(user.email)
+            api_v4.assign_user_organization_role(user_id, user.organization_role)
+            print("Updated organization role for {email} to {org_role}.".format(email=user.email,
+                                                                                org_role=user.organization_role))
+
+            # Queue the users to add for each team to economize requests
+            add_to_teams[user.team] = (user_id, user.team_role)
+
+    # Process team membership changes
+    for team, info in add_to_teams.items():
+        api_v4.assign_users_to_teams(team, info)
+        print("Updated team membership for '{}'".format(team))
+        for user_id, team_role in info:
+            u = users.user_for_id(user_id)
+            # is_member works because users info was obtained before making any changes so it depicts initial state.
+            action = "Updated team membership of " if u.is_member(team) else "Added membership of "
```

Comment:
```suggestion
            action = "Updated team membership of" if u.is_member(team) else "Added membership of"
```
(you're adding a space when printing it)